src/components/admin/UserProfileModal/components/header/ProfileModalBasicTextInfo.tsx

```diff
@@ -31,19 +25,17 @@ export const ProfileModalBasicTextInfo: React.FC<ProfileModalBasicTextInfoProps>
   const userStatus = world?.showUserStatus && world?.userStatuses?.[0];
 
   return (
-    <div
-      className={classNames("ProfileModalBasicTextInfo", containerClassName)}
-    >
-      <h3 className="ProfileModalBasicTextInfo__name italic">
+    <div className="ProfileModalBasicTextInfo flex flex-col">
```

Comment:
Same here for the `.scss` file classNames.

Reply:
I suggested to @sunny-viktoryia we move them to something like `data-bem="ProfileModalBasicTextInfo__name"`

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8